packages/cli/fragments/fragment-clojurescript/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "{{package_name}}",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "assets:build": "cp ./src/style.css ./public/",
```

Comment:
why is this necessary? can't you import the style inside .cljs file directly?

Reply:
Unfortunately, we cannot import styles into cljs

Reply:
then you need to move `base/src/style.css` to `_assets_/style.css` and add it to the `[files]` section in `_cta_manifest_` but you'd have to do the same for other fragments

Reply:
Oh, super! Thanks 👍